linux/entrypoint/pre_start.sh

```diff
@@ -12,6 +12,7 @@ $ENTRYPOINT_HOME/certs.sh
 
 props set org.codice.ddf.system.hostname $_app_hostname $APP_HOME/etc/system.properties
 props set org.codice.ddf.system.siteName $_app_hostname $APP_HOME/etc/system.properties
+props set org.codice.ddf.system.httpsPort $APP_HTTPSPORT $APP_HOME/etc/system.properties
```

Comment:
This variable either would need a default value set at the top of the script or to be enclosed in a conditional statement, otherwise if left unset it would nullify that property

try something like 
```bash
if [ -n "$HTTPS_PORT" ]; then
   props set org.codice.ddf.system.httpsPort $HTTPS_PORT $APP_HOME/etc/system.properties
if
```